src/Umbraco.PublishedCache.HybridCache/Persistence/DatabaseCacheRepository.cs

```diff
@@ -351,12 +358,20 @@ private void RebuildContentDbCache(IContentCacheDataSerializer serializer, int g
     }
 
     /// <summary>
-    /// Rebuilds the content database cache for media.
-    /// </summary>
-    /// <remarks>
+    /// Rebuilds the media database cache by clearing and repopulating the cache with the latest media data.
```

Comment:
This is a nice comment added, but if we are including, could we add the similar on `RebuildContentDbCache` and `RebuildMemberDbCache` for consistency please?